Makefile

```diff
@@ -20,6 +20,10 @@ GO_BUILD_PACKAGES :=$(strip \
 	./cmd/... \
 	$(wildcard ./tools/*) \
 )
+GO_TEST_PACKAGES :=$(strip \
```

Comment:
I'd do a separate target for those given how isolated they are and how much time they take
```
test-unit-kubectl: GO_TEST_PACKAGES:=./vendor/k8s.io/kubernetes/pkg/kubectl/...
test-unit-kubectl: test-unit
.PHONY: test-unit-kubectl
```

Reply:
No need for separate if the time diff between with and w/o is small:
with kubectl:
```
real	4m58.919s
user	13m48.102s
sys	1m26.500s
```
w/o kubectl:
```
real	4m49.915s
user	10m16.252s
sys	1m6.698s
```

Reply:
I can go for unified if the difference is negligible, the original estimate from you seemed much worse